<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Help</class>
 <widget class="QDialog" name="Help">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="Label_3">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Введіть ім'я</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="LineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Label_4">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Введіть прізвище</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="LineEdit_2"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Повідомлення</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Адміністратор:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Chancery L</family>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Вадим Щербина</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>13</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Help</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Help</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
